interview/domain/tts/tts_client.py

```
@@ -0,0 +1,6 @@
+from abc import ABC, abstractmethod
+from typing import List
+
+class PollyClient(ABC):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)